### PR TITLE
Update config to include colors

### PR DESCRIPTION
**Pull request by paolorau**
## Purpose

- Update the component to use provided colors

## Asana tasks

- [Update background colors in AssetPicker](https://app.asana.com/0/0/1203135611977724/f)

## Approach

- Created a new struct `AssetsPickerColors` that hold the different colors used by the component, it also provides default values (using semantic values for iOS 13+)
- Replace the old colors.

## Screenshots

- 

https://user-images.githubusercontent.com/2923778/195395432-77e0248d-dfce-4a3a-8f3d-52126df01d6f.MP4

### CODE REVIEW

**Review by Jmats17 — APPROVED** [review]
LGTM!

File: AssetsPickerViewController/Classes/Picker/AssetsPickerColors.swift [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    cellBackground: UIColor? = nil
+  ) {
+    self.defaultCheckmark = defaultCheckmark ?? defaultColors.defaultCheckmark
+    self.label = label ?? defaultColors.label
+    self.secondaryLabel = secondaryLabel ?? defaultColors.secondaryLabel
+    self.background = background ?? defaultColors.background
+    self.cellBackground = cellBackground ?? defaultColors.cellBackground
+  }
+  
+  struct defaultColors {
```
Comment (alanipce) [inline AssetsPickerViewController/Classes/Picker/AssetsPickerColors.swift]: Use pascal case for type definitions

File: AssetsPickerViewController/Classes/Picker/AssetsPickerColors.swift [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+//  AssetsPickerColors.swift
+//  
+//
+//  Created by Paolo Rau on 10/12/22.
+//
+import UIKit
+
+/// Class to hold color values for Picker elements.
+public struct AssetsPickerColors {
+  var defaultCheckmark: UIColor
```
Comment (alanipce) [inline AssetsPickerViewController/Classes/Picker/AssetsPickerColors.swift:12]: is the default checkmark used?

File: AssetsPickerViewController/Classes/Album/Controller/AssetsAlbumViewController.swift [hunk 3]
```diff
@@ -94,7 +94,7 @@ open class AssetsAlbumViewController: UIViewController {
     open override func loadView() {
         super.loadView()
         view = UIView()
-		view.backgroundColor = .ap_background
+        view.backgroundColor = AssetsManager.shared.pickerConfig.colors.background
```
Comment (alanipce) [inline AssetsPickerViewController/Classes/Album/Controller/AssetsAlbumViewController.swift]: can't you use the `pickerConfig` instance defined on the album controller

**Review by alanipce — COMMENTED** [review]
Left some general comments
- Would be ideal if we could avoid accessing the shared config to access the color definitions, on the controller we can access the config directly
- The defaultColors type is in camel case
- Work with Dmitry on updating the repo specs and using it on the Albert app

File: AssetsPickerViewController/Classes/Picker/AssetsPickerColors.swift (hunk 2 above)
Comment (paolorau) [reply AssetsPickerViewController/Classes/Picker/AssetsPickerColors.swift:12]: Good catch, I forgot to move `SSCheckMark.swift`

File: AssetsPickerViewController/Classes/Album/Controller/AssetsAlbumViewController.swift (hunk 3 above)
Comment (paolorau) [reply AssetsPickerViewController/Classes/Album/Controller/AssetsAlbumViewController.swift]: I've removed the usage of the shared object in the places where I have access to the picker config. 
@alanipce Should I take some time to remove all the usages? I.e for the cells I can quickly create a `configure` method and pass the proper data, please let me know.

**Review by alanipce — APPROVED** [review]

**Pull request merged**